Documentation/Books/Manual/Security/Auditing/AuditEvents.md

```diff
@@ -117,36 +120,36 @@ Documents
 ### Reading a single document
 
 ```
-2016-10-04 12:27:55 | server1 | user1 | database1 | 127.0.0.1:53699 | http basic | create document ok | /_api/document/collection1
+2016-10-04 12:27:55 | server1 | audit-documents | user1 | database1 | 127.0.0.1:53699 | http basic | create document in 'collection1' | ok | /_api/document/collection1
 ```
 
 ### Replacing a single document
 
 ```
-2016-10-04 12:28:08 | server1 | user1 | database1 | 127.0.0.1:53699 | http basic | replace document ok | /_api/document/collection1/21456?ignoreRevs=false
+2016-10-04 12:28:08 | server1 | audit-documents | user1 | database1 | 127.0.0.1:53699 | http basic | replace document 'collection1/21456' | ok | /_api/document/collection1/21456?ignoreRevs=false
 ```
 
 ### Modifying a single document
 
 ```
-2016-10-04 12:28:15 | server1 | user1 | database1 | 127.0.0.1:53699 | http basic | modify document ok | /_api/document/collection1/21456?keepNull=true&ignoreRevs=false
+2016-10-04 12:28:15 | server1 | audit-documents | user1 | database1 | 127.0.0.1:53699 | http basic | modify document 'collection1/21456' | ok | /_api/document/collection1/21456?keepNull=true&ignoreRevs=false
 ```
 
 ### Deleting a single document
 
 ```
-2016-10-04 12:28:23 | server1 | user1 | database1 | 127.0.0.1:53699 | http basic | delete document ok | /_api/document/collection1/21456?ignoreRevs=false
+2016-10-04 12:28:23 | server1 | audit-documents | user1 | database1 | 127.0.0.1:53699 | http basic | delete document 'collection1/21456' | ok | /_api/document/collection1/21456?ignoreRevs=false
 ```
 
 For example, if someones tries to delete a non-existing document, it will be logged as
 
 ```
-2016-10-04 12:28:26 | server1 | user1 | database1 | 127.0.0.1:53699 | http basic | delete document failed | /_api/document/collection1/21456?ignoreRevs=false
+2016-10-04 12:28:26 | server1 | audit-documents | user1 | database1 | 127.0.0.1:53699 | http basic | delete document 'collection/21456' | failed | /_api/document/collection1/21456?ignoreRevs=false
 ```
 
 Queries
 -------
 
 ```
-2016-10-06 12:12:10 | server1 | user1 | database1 | 127.0.0.1:54232 | http basic | query document | ok | for i in collection1 return i | /_api/cursor
+2016-10-06 12:12:10 | server1 | audit-documents | user1 | database1 | 127.0.0.1:54232 | http basic | query document | ok | for i in collection1 return i | /_api/cursor
```

Comment:
are the log topics in singular form (e.g. `audit-document`) but the 3rd column in the log entries in plural form (here: `audit-documents`)?

Reply:
Good catch.